lib/src/file_io_desktop_and_mobile.dart

```diff
@@ -25,7 +25,7 @@ Future<ByteData> loadFontFromDeviceFileSystem(String name) async {
 }
 
 Future<String> get _localPath async {
-  final directory = await path_provider.getApplicationSupportDirectory();
+  final directory = await getApplicationSupportDirectory();
```

Comment:
nit: when calling top-level functions that are defined in other files, I prefer to add an import prefix so I can tell which import the function is coming from.


something like

Above:
```dart
import 'application_support_directory.dart' as local_directory;
```

Here:
```dart
final directory = await local_directory.getApplicationSupportDirectory();
```

Reply:
Done. Thank you!